package.json

```diff
@@ -39,6 +39,7 @@
     "sinon": "1.10.3",
     "sinon-chai": "2.5.0",
     "bower": "1.3.12",
-    "grunt-sass": "0.17.0"
+    "grunt-sass": "0.17.0",
+    "underscore": "1.8.3"
   }
```

Comment:
We're not overriding underscore here are we?

Also how are these versions then used by the agent test runner?